### PR TITLE
Add seed Command

### PR DESCRIPTION
**Pull request by Commandcracker**
## Description
Requires #287 

## Checklist
Things need to be done before this Pull Request can be merged.

- [x] Code is well-formatted and adheres to project style guidelines: `cargo fmt`
- [x] Code does not produce any clippy warnings `cargo clippy`

### CODE REVIEW

File: pumpkin/src/command/commands/cmd_seed.rs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#[async_trait]
+impl CommandExecutor for PumpkinExecutor {
+    async fn execute<'a>(
+        &self,
+        sender: &mut CommandSender<'a>,
+        _server: &crate::server::Server,
+        _args: &ConsumedArgs<'a>,
+    ) -> Result<(), CommandError> {
+        let seed = &BASIC_CONFIG.seed;
```
Comment (Snowiiii) [inline pumpkin/src/command/commands/cmd_seed.rs:27]: That's only the seed from the Config, Which can be empty. If the seed is empty will will generate a random one later and store it in the `world`.

Comment (Commandcracker) [reply pumpkin/src/command/commands/cmd_seed.rs:27]: It's not in the World, it's in GeneratorInit which I can't really access.
IG ill make this into a draft until world gen is ready and has a seed.

Comment (Snowiiii) [reply pumpkin/src/command/commands/cmd_seed.rs:27]: You should now be able to get the seed in the level. See https://github.com/Snowiiii/Pumpkin/commit/b5a8fee243c6a4e57238bf863b57f716453e3c12

**Comment by Snowiiii** [comment]
Looks good now.
Thanks @Commandcracker

**Pull request merged**